### DISCUSSION TITLE
Need some help to create a RTSP Simple Server addon for Home Assistant

### DISCUSSION THREAD

**Discussion by SirLouen** [bluenviron/mediamtx#1143] [Questions]
I've been reading how to create a docker file with FFmpeg integrated 
https://github.com/aler9/rtsp-simple-server/discussions/278#discussioncomment-549104

But I don't have this clear to integrate this for an add-on module, which ultimately runs a docker command
There is a simple add-on already integration RTSP Simple server
https://github.com/fuatakgun/rtsp_simple_server

But obviously without FFmpeg. I would like to upgrade so, and also include a persistent YML outside the docker image, so I could be editing straight on the HA box

Here the dockerfile: https://github.com/fuatakgun/rtsp_simple_server/blob/main/Dockerfile

I am not sure if I could replace it as-is

**Comment by fuatakgun** [comment]
I do not think that respective docker file works at all, given that I am just pointing to image, no need to build from scratch. It is actually a leftover I need to clean sometime :)

**Comment by fuatakgun** [comment]
have you checked this? https://github.com/aler9/rtsp-simple-server/discussions/278#discussioncomment-549104

[a comment by an automated account is not shown]
